.github/pull_request_template.md

```diff
@@ -1,8 +1,10 @@
-## Describe your changes
+<!-- Describe your changes --> 
 
-## Issue ticket number and link
+<!-- Issue ticket number and link (if applicable) -->
+
+<!-- Testing -->
+- [ ] I have tested those changes 
+- [ ] I have added tests (skip if just adding/editing providers)
+
+<!-- Testing instructions (if applicable) -->
```

Comment:
arguably we could get rid of this one but @nalanj raised a good point where we often make the assumptions that the reviewers has all the context to tests the PR.